docs/arcadia-users-group/20240730-arcadiathemer/20240730-arcadiathemer.Rmd

```diff
@@ -0,0 +1,143 @@
+4. Install `arcadiathemeR` with the `remotes` package:
+
+    ```
+    # install.packages("remotes")
+    remotes::install_github("Arcadia-Science/arcadiathemeR")
+    ```
+5. Load the package to use in your scripts with `library(arcadiathemeR)`. When you first load the package it will print a message about whether the custom fonts have been accessed and loaded correctly.
+
+## Layering onto an existing `ggplot2` plot
+
+There are two main functions you will use to layer onto an existing `ggplot2` plot to create plots adhering to the Arcadia style guide and access palettes. These are the `theme_arcadia()` and `scale` functions. The particular `scale` function differs on whether you use `color` or `fill` to access colors. Here is how you use these functions without changing the default arguments:
+
+```{r, example_plot}
+library(ggplot2)
+library(arcadiathemeR)
+
+ggplot(data=mtcars, aes(x=hp, y=mpg, color=as.factor(cyl))) +
+  geom_point() +
+  theme_arcadia() +
+  scale_color_arcadia()
+```
+
+If you want to change the color palette used, you can access this with the `palette_name` argument in the `scale` function: 
+
+```{r, palette_change}
+ggplot(data=mtcars, aes(x=hp, y=mpg, color=as.factor(cyl))) +
+  geom_point() +
+  theme_arcadia() +
+  scale_color_arcadia(palette_name = "primary")
+```
+
+We have specific font specifications for whether the represented data is categorical or numerical. In the next plot where the x-axis is different categories, you can see where the font isn't specified correctly:
+```{r, fonts}
+ggplot(data=diamonds, aes(x=cut, fill=cut)) +
+  geom_bar() +
+  theme_arcadia() +
+  scale_fill_arcadia()
+```
+
+To use the correct font type, you can specify what type of data you have on each axis by using the `x_axis_type` or `y_axis_type` arguments in the `theme_arcadia()` function: 
+```{r, fonts_correct}
+ggplot(data=diamonds, aes(x=cut, fill=cut)) +
+  geom_bar() +
+  theme_arcadia(x_axis_type = "categorical") +
+  scale_fill_arcadia()
+```
+In addition to specifying which palette to use in the `scale` function, you can also reverse the colors of the scale used with `reverse=TRUE`. You can also use other `ggplot2` or `theme` specifications on top of these functions, such as moving the position of the legend or modifying the scales to remove whitespace between the axis lines and the bars:
+```{r, additional_layers}
+ggplot(data=diamonds, aes(x=cut, fill=cut)) +
+  geom_bar() +
+  theme_arcadia(x_axis_type = "categorical") +
+  scale_fill_arcadia(palette_name = "secondary", reverse = TRUE) +
+  scale_y_continuous(expand=c(0,0)) + # removes whitespace between axis and bars
+  theme(legend.position = "bottom")
+```
+In addition to reversing the order of the colors used in the palette, you can select different indices of colors from the palettes within the `scale` function with the `start` and `end` arguments:
```

Comment:
Line breaks should be added before all of the paragraphs abutting ``` marks so that the Markdown displays correctly

```suggestion

In addition to reversing the order of the colors used in the palette, you can select different indices of colors from the palettes within the `scale` function with the `start` and `end` arguments:
```

Reply:
these should all be fixed now